.github/workflows/reusable_teleport_operational_procedure.yml

```diff
@@ -0,0 +1,276 @@
+              id: create-namespaces
+              env:
+                  AWS_SECRET_ACCESS_KEY_ES: ${{ steps.secrets.outputs.S3_BACKUP_SECRET_KEY }}
+                  AWS_ACCESS_KEY_ES: ${{ steps.secrets.outputs.S3_BACKUP_ACCESS_KEY }}
+              working-directory: ./test
+              run: |
+                  set -euxo pipefail
+                  go test --count=1 -v -timeout 9m -run TestClusterPrerequisites
+
+            - name: Label namespaces and secrets
+              run: |
+                  set -euxo pipefail
+
+                  label_resources() {
+                    local namespace="$1"
+                    kubectl label namespace "$namespace" ${{ env.LABELS }} --overwrite
+                    local secrets
+                    secrets="$(kubectl get secrets -n "$namespace" -o jsonpath='{.items[*].metadata.name}')"
+                    for secret in $secrets; do
+                      kubectl label secret "$secret" -n "$namespace" ${{ env.LABELS }} --overwrite
+                    done
+                  }
+
+                  # Combine cluster-0 and cluster-1 arrays into one comma-separated list
+                  IFS=',' read -r -a ALL_NAMESPACES <<< "${CLUSTER_0_NAMESPACE},${CLUSTER_1_NAMESPACE}"
+                  for namespace in "${ALL_NAMESPACES[@]}"; do
+                    label_resources "$namespace"
+                  done
+
+            - name: Deploy dual region Camunda
+              id: deploy-camunda
+              working-directory: ./test
+              run: |
+                  set -euxo pipefail
+                  go test --count=1 -v -timeout 30m -run TestAWSDeployDualRegCamunda
+
+            - name: Failover New - ${{ inputs.helmChartVersion }}
+              working-directory: ./test
+              timeout-minutes: 21
+              run: |
+                  set -euxo pipefail
+                  go test --count=1 -v -timeout 20m -run TestAWSDualRegFailover_8_6_plus
+
+            - name: Set start timestamp for Failback
+              id: failback-start
+              run: |
+                  set -euxo pipefail
+                  printf 'timestamp=%(%s)T\n' >> "$GITHUB_OUTPUT"
+            - name: Failback New - ${{ inputs.helmChartVersion }}
+              working-directory: ./test
+              timeout-minutes: 46
+              run: |
+                  set -euxo pipefail
+                  go test --count=1 -v -timeout 45m -run TestAWSDualRegFailback_8_6_plus
+            - name: Calculate Failback duration
+              run: |
+                  set -euxo pipefail
+                  printf -v now '%(%s)T'
+                  duration=$((now - ${{ steps.failback-start.outputs.timestamp }}))
+                  echo $duration
+                  if [ "$duration" -gt "1500" ]; then
+                  echo "::error ::Failback of ${{ inputs.helmChartVersion }} is taking longer than 25 minutes"
+                  fi
+
+            - name: Delete namespaces
+              if: ${{ always() }}
```

Comment:
just as side note, you can also just do `if: always()`

[reference](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#about-expressions)